common/oauthTokenManager.go

```diff
@@ -343,8 +442,23 @@ func (uotm *UserOAuthTokenManager) HasCachedToken() (bool, error) {
 	return uotm.credCache.HasCachedToken()
 }
 
-// RemoveCachedToken delete all the cached token.
+// RemoveCachedToken deletes the token and the AuthenticationRecord as part of the logout command execution.
+// Returns an error if encountered during file deletion or checking.
 func (uotm *UserOAuthTokenManager) RemoveCachedToken() error {
+	filePath := filepath.Join(AzcopyJobPlanFolder, defaultAuthFileName)
+
+	// Check if the file exists
+	if _, err := os.Stat(filePath); err == nil {
+		// File exists, so delete it
+		if err := os.Remove(filePath); err != nil {
+			return fmt.Errorf("error deleting file: %v", err)
+		}
+		fmt.Println("File deleted successfully.")
```

Comment:
These logs do not give any indication on what is happening or what has failed and on top of that these are going to user console. Imagine user getting an error message "file does not exist.". What is user suppose to imply from this message?

Reply:
Done in the latest commit.